README.md

```diff
@@ -106,7 +106,7 @@ if err = service.Stop(); err != nil {
 In order to handle events, you can call the Events function inside a goroutine.
 
 ```
-dockercloud.StreamUrl = "wss://ws.cloud.docker.com:443/v1/"
+dockercloud.StreamUrl = "wss://ws.cloud.docker.com"
```

Comment:
Right. And I think it is better to change the code to accommodate both cases, w/ and w/o "/"